src/main/java/org/commcare/resources/model/ResourceTable.java

```diff
@@ -413,13 +414,17 @@ private void findResourceLocationAndInstall(Resource r,
                     // Continue until no resources can be found.
                 } catch (UnreliableSourceException use) {
                     unreliableSourceException = use;
+                } catch (UnresolvedResourceException ure) {
```

Comment:
Note: I believe changes to this call stack should also generally be replicated to line 390 above